py/parse_and_notify.py

```diff
@@ -193,9 +272,17 @@ def build(cls, raw_content: Mapping[str, Any]) -> UpdatedTopic:
 
 
 def main():
-    token = os.getenv("DATADOG_API_KEY")
-    assert token is not None, "No Datadog token in DATADOG_API_KEY env var"
-    notifier = Notifier(datadog_api_key=token)
+    dd_token = os.getenv("DATADOG_API_KEY")
+    slack_secret = os.getenv("KAFKA_CONTROL_PLANE_WEBHOOK_SECRET")
```

Comment:
Why is one named after the control plane and the other is not ?

Reply:
The DD one is an API key, `slack_secret` is the signing secret for the KCP webhook channel in eng-pipes.
I'll add a comment

Reply:
No, please do not add a comment to explain the naming convention. 
Naming convention are meant so that you do not have to explain them and they serve as a way to explain what you are doing. 

Please leave it as it is.